libopenage/renderer/opengl/indexbuffer.cpp

```diff
@@ -0,0 +1,57 @@
+// Copyright 2015-2017 the openage authors. See copying.md for legal info.
+
+#include "indexbuffer.h"
+
+#include "../../error/error.h"
+
+
+namespace openage {
+namespace renderer {
+namespace opengl {
+
+GlIndexBuffer::GlIndexBuffer(size_t size, GLenum usage)
```

Comment:
This looks like effectively the same code as for `GlBuffer`, so we could add an enum for the buffer type there and remove this class.

Reply:
That would be a better idea